src/common/mfu_flist_copy.c

```diff
@@ -497,23 +523,25 @@ static int mfu_copy_timestamps(
     return rc;
 }
 
-static int mfu_copy_close_file(mfu_copy_file_cache_t* cache)
+static int mfu_copy_close_file(mfu_copy_file_cache_t* cache, mfu_file_t* mfu_file)
 {
     int rc = 0;
 
     /* close file if we have one */
     char* name = cache->name;
     if (name != NULL) {
         int fd = cache->fd;
-
+#ifdef DAOS_SUPPORT
+        dfs_obj_t* obj = cache->obj;
+#endif
         /* if open for write, fsync */
         int read_flag = cache->read;
-        if (! read_flag) {
+        if (! read_flag && mfu_file->type == POSIX) {
```

Comment:
Does DAOS have any kind of sync operation that would apply here?

Reply:
@adammoody currently DAOS doesn't have a dfs_sync that is supported. It says it will be in the future though, but not sure when that will be.

Reply:
@adammoody it is not yet supported as part of the DAOS filesystem API for POSIX yet at least.